packages/grid/_modules_/grid/components/menu/GridMenu.tsx

```diff
@@ -19,6 +20,17 @@ type MenuPosition =
   | 'top'
   | undefined;
 
+const useStyles = makeStyles(
+  () => ({
+    root: {
+      '& .MuiDataGrid-gridMenuList:focus': {
```

Comment:
We don't need to mention `:focus` it's shorter without

```suggestion
      '& .MuiDataGrid-gridMenuList': {
```